torchbenchmark/models/pytorch_CycleGAN_and_pix2pix/__init__.py

```diff
@@ -35,16 +35,18 @@ def __init__(self, test, device, batch_size=None, extra_args=[]):
         results_arg = f"--results_dir {results_dir}"
         data_root = os.path.join(DATA_PATH, "pytorch_CycleGAN_and_pix2pix_inputs")
         device_arg = ""
+        device_type = f"--device_type {self.device}"
```

Comment:
Please follow the code style and rename the variable with `arg` suffix - `devict_type_arg` or refine the code a little bit
```python
        device_arg = f"--device_type {self.device} "
        if self.device == "cpu":
            device_arg += "--gpu_ids -1"
        else:
            device_arg + = "--gpu_ids 0"
```

Reply:
The var "device_type" is renamed "device_type_arg".